include/aikido/perception/RcnnMopedModule.hpp

```diff
@@ -0,0 +1,80 @@
+#ifndef AIKIDO_PERCEPTION_RCNNPOSEMODULE_HPP_
+#define AIKIDO_PERCEPTION_RCNNPOSEMODULE_HPP_
+
+#include <string>
+#include <dart/dart.hpp>
+#include <ros/ros.h>
+#include <tf/transform_listener.h>
+#include <aikido/io/CatkinResourceRetriever.hpp>
+#include <aikido/perception/ObjectDatabase.hpp>
+#include <aikido/perception/PerceptionModule.hpp>
+
+namespace aikido {
+namespace perception {
+
+/// Instantiates the \c PerceptionModule specifically for RcnnPose.
+///
+/// It receives input from the RcnnPose running in a separate process,
+/// by subscribing to a visualization_msg::MarkerArray ROS topic published by
+/// the RcnnPose node. It uses a \c ObjectDatabase to resolve each marker to a
+/// \c dart::common::Uri, and updates the environment which is a list of \c
+/// dart::dynamics::SkeletonPtr.
+class RcnnPoseModule : public PerceptionModule
+{
+public:
+  /// Construct an RcnnPose receiver that subscribes to the specified topic
+  /// where objects' pose information is being published as a
+  /// \c visualization_msg::MarkerArray, uses a database loader for
+  /// configuration information related to tags.
+  ///
+  ///	\param[in] nodeHandle The node handle to be passed to the detector
+  ///	\param[in] markerTopic The name of the topic on which objects' pose
+  /// information is being published
+  ///   \param[in] resourceRetriever A CatkinResourceRetriever for resources
+  /// related to config files and models
+  ///	\param[in] referenceFrameId The desired reference frame for the
+  /// object pose
+  ///	\param[in] referenceLink A link on the robot with respect to which the
+  /// pose is transformed
+  RcnnPoseModule(
+      ros::NodeHandle nodeHandle,
+      std::string markerTopic,
+      std::shared_ptr<ObjectDatabase> configData,
```

Comment:
This parameter is missing documentation.

Reply:
added documentation for `configData`